hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -280,6 +280,14 @@ private FlinkOptions() {
           + "usually with delta time compaction strategy that is long enough, for e.g, one week;\n"
           + "2) changelog mode is enabled, this option is a solution to keep data integrity");
 
+  // this option is experimental
+  public static final ConfigOption<Boolean> READ_STREAMING_SKIP_CLUSTERING = ConfigOptions
+          .key("read.streaming.skip_clustering")
+          .booleanType()
+          .defaultValue(false)
+          .withDescription("Whether to skip clustering instants for streaming read,\n"
+              + "to avoid reading duplicates");
```

Comment:
Could you add the cases which cause reading duplicates in the description?

Reply:
Yeah, that's needed, can you add more cases for clustering here ? You can add UTs in `TestInputFormat`.

Reply:
@danny0405, ok. I will do adding the UTs.